jestconfig.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Fun Fact: We have two jest configs floating around in our repo. This one is used when running jest in the individual packages, one in package.json is run when running jest on the lerna repo overall?
Two questions,
1. Does running jest at the main repo have an issue on PC?
2. Could you consolidate these configs (I think you could just delete the config from package.json)

![rockfact](https://user-images.githubusercontent.com/1588747/40856826-ce411a9c-659e-11e8-8591-ce5bd1b4f947.png)

Reply:
1. Running Jest on the main repo has no issue on PC.

2. Looking into it now